docs/theories/datatypes.rst

```diff
@@ -183,19 +183,19 @@ a `cvc5::api::Solver solver` object.
+|                    |                                        | ``Sort s = solver.mkTupleSort({s_int, s_int});``                                                                                |
 |                    |                                        |                                                                                                                                 |
 |                    |                                        | ``Term t = solver.mkConst(s, "t");``                                                                                            |
 +--------------------+----------------------------------------+---------------------------------------------------------------------------------------------------------------------------------+
-| Tuple Constructor  | ``(mkTuple <Term_1>, ..., <Term_n>)``  | ``Sort s = solver.mkTypleSort(sorts);``                                                                                         |
+| Tuple Constructor  | ``(mkTuple <Term_1>, ..., <Term_n>)``  | ``Sort s = solver.mkTupleSort(sorts);``                                                                                         |
 |                    |                                        |                                                                                                                                 |
 |                    |                                        | ``Datatype dt = s.getDatatype();``                                                                                              |
 |                    |                                        |                                                                                                                                 |
 |                    |                                        | ``Term c = dt[0].getConstructor();``                                                                                            |
 |                    |                                        |                                                                                                                                 |
 |                    |                                        | ``Term t = solver.mkTerm(Kind::APPLY_CONSTRUCTOR, {c, <Term_1>, ..., <Term_n>});``                                              |
 +--------------------+----------------------------------------+---------------------------------------------------------------------------------------------------------------------------------+
-| Tuple Selector     | ``((_ tupSel i) t)``                   | ``Sort s = solver.mkTypleSort(sorts);``                                                                                         |
+| Tuple Selector     | ``((_ tuple_select i) t)``                   | ``Sort s = solver.mkTupleSort(sorts);``                                                                                         |
```

Comment:
The spacing changes will break the table, I guess

Reply:
Fixed.